server/mediapublic/mediapublic/views.py

```diff
@@ -594,8 +596,138 @@ def blogs_comment_by_id(request):
     id = request.matchdict['id']
     if not request.method in METHODS:
         resp = {'error': 'Method Not Allowed'}
+        resp, status = METHODS[request.method](request, id, cls)
         status = 405
     else:
-        resp, status = METHODS[request.method](request, id, cls)
+    request.response.status = status
+    return resp
+
+########### PLAYLISTS
+# [GET]
+@view_config(route_name='playlists', renderer='json')
+def station_playlists(request):
+    cls = Playlists
+    resp = []
+    if request.method == "GET":
+        resp = [i.to_dict() for i in cls.get_all()]
+    else:
+        resp = {'error': 'Method Not Allowed'}
+        request.response.status = 405
+    return resp
+
+# [GET, POST]
+@view_config(route_name='user_playlists', renderer='json')
+def user_playlists(request):
+    cls = Playlists
+    resp = []
+    if request.method == "GET":
+        resp = [i.to_dict() for i in cls.get_by_user_id(person_id)]
+    elif request.method == "POST":
+        person_id = request.matchdict['uid']
+        keys = {}
+        playlist, status = do_post(request, keys, cls)
+        request.response.status = status
+        request.response.status = 303
+        # Which one should be used here? Maybe only redirect if == 200?
+        request.route_url('edit_user_playlist', uid=person_id, pid=playlist['id'])
+    else:
+        resp = {'error': 'Method Not Allowed'}
+        request.response.status = 405
+    return resp
+
+# [GET]
+@view_config(route_name='edit_user_playlist', renderer='json')
```

Comment:
The Python Pyramid app is just for REST end points ( for now ).  So all things are just JSON payloads with `GET, POST, PUT, DELET` methods.  There are no pages returned ( so the edit page where you drag things around would not be defined here, it would be a virtual # page on the GUI ).  Additionally, the renderer for the end-point is set to json, so I'm not really sure what would be returned.